app/messaging/create-message.js

```diff
@@ -0,0 +1,10 @@
+const createMessage = (body, type, options) => {
+  return {
+    body,
+    type,
+    source: 'ffc-pay-event-publisher',
```

Comment:
Should we set the source as the service using the package.  Otherwise all our events will have the same source and we won't know the origin